integration_tests/tests/forseti/controls/scanner.rb

```diff
@@ -40,6 +40,10 @@
                expect(command("forseti scanner run").stdout).to match /Scan completed/
            end
 
+           it "should be visible in the database" do
+               expect(command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(*) from violations where violation_type = 'IAM_POLICY_VIOLATION';\"").stdout).to match /2/
```

Comment:
Is there a way to tune the db query so that it exactly find the violation from the bucket that's added here?  Otherwise, this test will break, when other resources with IAM violations are added.

Reply:
Bucket names are randomized, so a check on the name of the bucket cannot be added, but a check to see if the resource is `bucket` can be added. This will prevent breakage when resources other than`bucket` are added.

Reply:
Can you please add the check to for the resource is bucket into the query?

Reply:
My point is that the query is currently very generic, and needs to be more scoped to the bucket violations that you are adding here.  Otherwise, this test will break again, when new IAM violations are added in the future.
```
\"select count(*) from violations where violation_type = 'IAM_POLICY_VIOLATION';\"
```

Reply:
Will merge this in for now, and improve this when more violations are added.